Update Button Overview (#646)
telerik/blazor-docs: components/button/disabled-button.md

---
title: Disabled Button
page_title: Disabled Button
description: Learn how to configure a disabled Blazor Button component by Telerik UI.

> slug: disabled-button

Reviewer: Don't we always start with the component name?

```suggestion
slug: button-disabled
```

tags: telerik,blazor,disabled,button,
published: True
position: 5
---

# Disabled Button

Sometimes specific buttons in an application must be temporarily disabled. To control the enabled state of the component, use the `Enabled` Boolean attribute.

> The following example demonstrates how to enable and disable the Button.
> 
> To render a disabled button, set its `Enabled` attribute to `false`.
> 
> >caption Disabled Telerik Button
> 
> ````CSHTML
> <TelerikButton Enabled="false">Disabled Button</TelerikButton>
> ````

Reviewer: The example should demonstrate "how to enable and disable".

```suggestion
The following example demonstrates how to enable and disable the Button.

>caption Toggle Telerik Button Enabled State

````CSHTML
<p>
    <label>
        <TelerikCheckBox @bind-Value="@ButtonIsEnabled" /> Toggle Button State
    </label>
</p>

<TelerikButton Enabled="@ButtonIsEnabled">@ButtonText</TelerikButton>

@code {
    bool ButtonIsEnabled { get; set; } = false;
    string ButtonText => ButtonIsEnabled ? "Enabled Button" : "Disabled Button";
}
````
```
